src/config.rs

```diff
@@ -0,0 +1,49 @@
+use std::fs::File;
+use std::io::BufReader;
+use std::ops::Deref;
+
+use pyo3::exceptions;
+use pyo3::prelude::*;
+
+use sticker_utils::{Config, TomlRead};
+
+/// Tagger configuration.
```

Comment:
~~~rust
/// Config(path)
/// --
///
/// Tagger configuration.
~~~

gives a signature via the `inspect` module:

~~~python
In [3]: inspect.signature(Config)                                                                                                                             
Out[3]: <Signature (path)>
~~~

or via IPython: 
~~~python
In [3]: sticker.Config?                                                                                                                                               
Init signature: Config(path)
Docstring:      Tagger configuration.
Type:           type
Subclasses:     
~~~

Reply:
Nice find! So it works when you put it on the class rather than the constructor.

Reply:
Added, though I called it `Config(file)`, following the signature of `open`.

Reply:
But at least `help(Config)` does show the signature on top. Having a signature on `__new__` would be nicer of course.

Reply:
For me, it only displays the super type:

~~~python
In [1]: from sticker import Config                                                                                                                            

In [2]: help(Config)         

Help on class Config in module builtins:

class Config(object)
 |  Tagger configuration.
 |  
 |  Methods defined here:
 |  
 |  __new__(*args, **kwargs) from builtins.type
 |      Create and return a new object.  See help(type) for accurate signature.
~~~

Reply:
Interesting, this is what I get:

```
>>> help(sticker.Config)
Help on class Config in module builtins:

class Config(object)
 |  Config(file)
 |  
 |  Tagger configuration.
 |  
 |  Static methods defined here:
 |  
 |  __new__(*args, **kwargs) from builtins.type
```

```
% python --version
Python 3.7.4
```

Reply:
Also works in iPython (which you seem to use?).

Reply:
Am on 3.6.5, seems it was added after that release:

https://github.com/python/cpython/commit/ccb5f3cee9ebb5f20e33f31432a46579f8e9bf8e#diff-cbbbf1954ca2586f5d1d6241674fab9a

Reply:
That's a nice change then!